raiden/tests/unit/test_blockchain_events_handler.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from raiden_contracts.constants import ChannelEvent
+
+from raiden.blockchain.events import Event
+from raiden.blockchain_events_handler import create_update_transfer_state_change
+from raiden.tests.utils import factories
+from raiden.transfer.state_change import ContractReceiveUpdateTransfer
+
+
+@pytest.fixture
+def container():
```

Comment:
I'd rather have this called `chain_state`

Reply:
I'd rather not call it `chain_state` as it's actually a dataclass containing a `chain_state` and other items. I'll go for `chain_state_setup` or something.